### PR TITLE
Profile to LinkFree

### PR DESCRIPTION
**Pull request by RonenTGreat**
## Changes proposed

 Added my profile to the LinkFree

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/958"></a>

### CODE REVIEW

**Review by subash7038 — APPROVED** [review]

File: public/data/ronenhammnd.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Ronen_T_G",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ronen-hammond",
+      "icon": "linkedIn"
```
Comment (syedsohan) [inline public/data/ronenhammnd.json:20]:
```suggestion
      "icon": "linkedin"
```

Comment (Panquesito7) [inline public/data/ronenhammnd.json:20]:
```suggestion
      "icon": "linkedin"
```

File: public/data/ronenhammnd.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Ronen Hammond",
+  "type": "personal",
+  "bio": "I am a student studying Bsc. Information Technology. Currently focusing on web development. I am open to collaborating on any open source project to increase my skill.",
```
Comment (syedsohan) [inline public/data/ronenhammnd.json:4]:
```suggestion
  "bio": "I am a student studying B.Sc. in Information Technology. Currently focusing on web development. I am open to collaborating on any open source project to increase my skill.",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**